Allow to use bessel and complex functions without undefining

From: Alexey Pavlov <alexpux@gmail.com>

From 58ff03526ba9a0e5ebe2e318b54fdbe993d5abbd Mon Sep 17 00:00:00 2001
Subject: [PATCH 1/3] Allow to use bessel and complex functions without
 undefining
---
 mingw-w64-headers/crt/math.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mingw-w64-headers/crt/math.h b/mingw-w64-headers/crt/math.h
index 3754dc64e..4bdd550d5 100644
--- a/mingw-w64-headers/crt/math.h
+++ b/mingw-w64-headers/crt/math.h
@@ -253,7 +253,7 @@ extern "C" {
 #define EDOM 33
 #define ERANGE 34
 
-#ifndef __STRICT_ANSI__
+#if !defined(__STRICT_ANSI__) || defined(_POSIX_C_SOURCE) || defined(_POSIX_SOURCE) || defined(_XOPEN_SOURCE) || defined(_GNU_SOURCE) || defined(_BSD_SOURCE)
 
 #ifndef _COMPLEX_DEFINED
 #define _COMPLEX_DEFINED
